urllib3/contrib/securetransport.py

```diff
@@ -91,6 +91,9 @@
 # individual cipher suites. We need to do this becuase this is how
 # SecureTransport wants them.
 CIPHER_SUITES = [
+    SecurityConst.TLS_AES_128_GCM_SHA256,
+    SecurityConst.TLS_AES_256_GCM_SHA384,
+    SecurityConst.TLS_CHACHA20_POLY1305_SHA256,
```

Comment:
Can we keep this list in the same order for both OpenSSL and SecureTransport, please?